File: Backend/Proyecto_ssdd/src/main/java/um/ssdd/proyecto_ssdd/Spring/Services/DTOs/UsuarioPut.java
package um.ssdd.proyecto_ssdd.Spring.Services.DTOs;

import java.io.Serializable;

public class UsuarioPut implements Serializable {

	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	private String id;
	private String user;
	private String password;
	private String nombre;
	private String apellidos;

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getUser() {
		return user;
	}

	public void setUser(String user) {
		this.user = user;
	}

	public String getPassword() {
		return password;
	}

	public void setPassword(String password) {
		this.password = password;
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public String getApellidos() {
		return apellidos;
	}

	public void setApellidos(String apellidos) {
		this.apellidos = apellidos;
	}

}

File: Frontend/src/app/DTOs/UsuarioPut.ts
export class UsuarioPut {

  id: string;
  user: string;
  password: string;
  nombre: string;
  apellidos: string;

  constructor(id: string, user:string, nombre: string, apellidos: string) {
    this.id = id;
    this.user = user;
    this.nombre = nombre;
    this.apellidos = apellidos;
    this.password = "";

  }
}

